mock_service.go

```diff
@@ -31,33 +32,58 @@ type MockEndpoint struct {
 	// RequestBody     string            `json:"requestBody"`
 }
 
-func New(createMockEndpoint string) *MockService {
+type MockEndpointSeries []MockEndpoint
+
+func New(mockCreatorEndpoint string) *MockService {
 	return &MockService{
-		createMockEndpoint: createMockEndpoint,
-		mockEndpoints:      map[string]map[string]MockEndpoint{},
+		mockCreatorEndpoint:  mockCreatorEndpoint,
```

Comment:
I was, but a thought it is a little long... sone I use Creator. Although Registration is more consistent with the other context.